### ISSUE TITLE
w.i.p 

### ISSUE THREAD

**Issue by migom6** [grayhoundquiz/grayhoundquiz.org#1]
website work in progress, ready by tomorrow .

**Comment by zombiesingh** [comment]
@migom6 When the website is ready push it in this repo

**Comment by migom6** [comment]
is the brochure done yet? upload in this repo too.

**Comment by zombiesingh** [comment]
Nandan is on it.

**Comment by migom6** [comment]
it will be helpful for me to write the content of the website

**Comment by zombiesingh** [comment]
No, brochure is not done. Upload atleast the landing page asap.

**Comment by zombiesingh** [comment]
Put up the template, we can the content later. Just put up the template.

**Comment by zombiesingh** [comment]
Page live kar de, gryhound ke domain pe daal de
